aot-cli/build.gradle.kts

```diff
@@ -28,7 +28,7 @@ val testAotRuntime by configurations.creating {
 
 dependencies {
     implementation(projects.aotApi)
-    implementation(mn.picocli)
+    implementation(libs.picocli)
```

Comment:
Ok, the PicoCLI BOM is missing as an import in core 4.0.0-SNAPSHOT

Investigating

Reply:
Right, so this project doesn't pull in the BOM, in order that we minimize the dependencies that will end up on the users classpath after AOT

Previously, we used to declare the version in the catalog for core, but we did away with that as part of the effort to stop repeating versions across multiple projects

After discussion (and investigation 🙏) by myself and @melix we decided the best thing here was to pull in the bom for the aot-cli module so that we use the picocli version declared there, without having to repeat the version here

@wetted ☝️ fixed in https://github.com/micronaut-projects/micronaut-aot/pull/130/commits/0a5d0a1a40936838e1bc809bda2d62386b8ff698 and https://github.com/micronaut-projects/micronaut-aot/pull/130/commits/40750f21b98fc6d42b5cdd6afe21c8f36df53dbe. PTAL 👍